shared/api/init.go

```diff
@@ -48,6 +52,21 @@ type InitNetworksProjectPost struct {
 	Project string
 }
 
+// InitStorageVolumesProjectPost represents the fields of a new LXD storage volume along with its associated pool.
+//
+// swagger:model
+//
+// API extension: preseed.
```

Comment:
This should be init_preseed_storage_volumes as it was introduced with that API extension.

Also please can you break the changes to shared/api into its own commit so they can be backported independently of the feature.